.gitignore

```diff
@@ -20,4 +20,4 @@ yarn-error.log*
 
 
 #TODO add .env to your gitignore
-
+package-lock.json
```

Comment:
You actually aren't supposed to gitignore this - https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5